### PR TITLE
Added Liberica JDK 8 (x64).

### PR DESCRIPTION
**Pull request by den1ska**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/655)

### CODE REVIEW

File: manifests/BellSoft/LibericaJDK/8.0.252.9.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Id: BellSoft.LibericaJDK
+Name: Liberica JDK 8
+Version: 8.0.252.9
+Publisher: BellSoft
+Homepage: https://bell-sw.com
+License: GPLv2 with exceptions
```
Comment (jamierocks) [inline manifests/BellSoft/LibericaJDK/8.0.252.9.yaml:6]: This should be `GPL 2 with Classpath Exception`.

**Review by jamierocks — CHANGES_REQUESTED** [review]

File: manifests/BellSoft/LibericaJDK/8.0.252.9.yaml (hunk 1 above)
Comment (den1ska) [reply manifests/BellSoft/LibericaJDK/8.0.252.9.yaml]: Thank you. Fixed.

**Comment by jamierocks** [comment]
When this installer is run, does it do so silently?

If not, the installer probably provides switches that allow such behaviour.

**Comment by den1ska** [comment]
> When this installer is run, does it do so silently?
> 
> If not, the installer probably provides switches that allow such behaviour.

I've just added switches to run the installer in the non-interactive mode.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**